### PR TITLE
Add support for valuetype configure flag

### PR DESCRIPTION
**Pull request by tajila**
Add support for valuetype configure flag

Signed-off-by: Tobi Ajila <atobia@ca.ibm.com>

### CODE REVIEW

File: runtime/include/j9cfg.h.in [hunk 1]
```diff
@@ -277,8 +277,7 @@ extern "C" {
 #cmakedefine J9VM_THR_LOCK_RESERVATION
 #cmakedefine J9VM_THR_PREEMPTIVE
 #cmakedefine J9VM_THR_SMART_DEFLATION
-
-#undef J9VM_OPT_VALHALLA_VALUE_TYPES
+#cmakedefine J9VM_OPT_VALHALLA_VALUE_TYPES
```
Comment (keithc-ca) [inline runtime/include/j9cfg.h.in]: Please move this into its place in alphabetical order above.

**Comment by tajila** [comment]
ci falure is

```
Resolving api.adoptopenjdk.net (api.adoptopenjdk.net)... 104.17.159.60, 104.17.158.60

Connecting to api.adoptopenjdk.net (api.adoptopenjdk.net)|104.17.159.60|:443... connected.

HTTP request sent, awaiting response... 404 Not Found

2020-03-23 21:43:42 ERROR 404: Not Found.
```

Ill trigger another run

File: runtime/include/j9cfg.h.in [hunk 2]
```diff
@@ -247,6 +247,7 @@ extern "C" {
 #cmakedefine J9VM_OPT_USE_FFI
 #cmakedefine J9VM_OPT_USE_FFI_ONLY
 #cmakedefine J9VM_OPT_VALHALLA_MVT
```
Comment (keithc-ca) [inline runtime/include/j9cfg.h.in]: `J9VM_OPT_VALHALLA_MVT` doesn't appear to be used anywhere; while you're here, perhaps you should remove this definition and the option in `runtime/cmake/options.cmake`.

**Comment by keithc-ca** [comment]
Jenkins test sanity zlinux jdk14

**Comment by keithc-ca** [comment]
Jenkins test sanity xlinux jdk14

**Comment by keithc-ca** [comment]
Even though it wasn't reported here, the [travis build](https://travis-ci.org/github/eclipse/openj9/builds/666356579) passed.

**Pull request merged**